_posts/2015-06-10-agile-fluency.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "ePages turns agile"
+date: "2015-06-10 07:28:00"
+icon: refresh
+tags: agile scrum company
+categories: scrum
+authors: ["Anja Bärike"]
+---
+
+Did you ever wonder how companies that are going the agile way are developing over time? And how long it actually takes to get from level to level? Or how many companies achieve certain levels of agility?
+
+Here are all the answers:
+
+{% image blog-agile-fluency.jpg %}{% endimage %}
+
+The picture shows the so called Agile Fluency Model. The numbers are based on questions that had been asked to companies who changed to an agile methodology some years ago. The most impressive numbers are probably, that it takes really long to achieve two or three stars. It is astonishing that only 5% of all companies who decide to go the agile way achieve a three star model.
+
+ePages is right on track between the first and the second star. It fits the time frame and we are on a good way to achieve the second star. We are definitely willing to belong to the 30% of companies who achieve that level. And we will not give up after that and fight on to be part of the only 5% that get to three stars!
```

Comment:
Headline might be a bit misleading, since the article is not so much about us but more about general agile stuff. But it's ok I guess